qemu/tests/virtio_fs_share_data.py

```diff
@@ -578,8 +578,8 @@ def __file_check(file, os_type):
                                       "the link file content in host is %s." %
                                       (content, link_guest, link_host))
                         # check the file in folder link
-                        folder_link = folder_link + "\\" + test_file
-                        link_guest = __file_check(folder_link, 'win_guest')
+                        folder_link_guest = folder_link + "\\" + test_file
+                        link_guest = __file_check(folder_link_guest, 'win_guest')
```

Comment:
Modify the parameter's name can fix the problem,right?
Could you explain more about how it works?Thanks!

Reply:
> Modify the parameter's name can fix the problem,right? Could you explain more about how it works?Thanks!

You can check the code logically.
 folder_link = folder_link + "\\" + test_file  folder_link is changed.

Reply:
Are you mean you also need update `"\\" `to `"\"`？

Reply:
NO,  folder_link = folder_link + "\\" + test_file
The variable folder_link is changed, so use "folder_link_guest = folder_link + "\\" + test_file" to replace it.

Reply:
Got it, thanks.
Could you update the commit message to make it more clear?Currently commit message(`The path on host isn't correct when vm is windows os`) seems not explain why we need to update the parameter's name, what do you think?

Reply:
Updated, thanks.